overlord/snapstate/snapstate.go

```diff
@@ -774,7 +774,8 @@ func updateManyFiltered(ctx context.Context, st *state.State, names []string, us
 		return nil, nil, err
 	}
 
-	updates, stateByInstanceName, ignoreValidation, err := refreshCandidates(ctx, st, names, user, nil)
+	refreshFlags := &store.RefreshOptions{IsAutoRefresh: flags.IsAutoRefresh}
```

Comment:
this should be refreshOpts, not Flags.

Is this bit of change tested?  not obvious looking around

Reply:
Yes, via SnapAction on the autoRefreshStore. Not super obvious